package.json

```diff
@@ -55,6 +55,7 @@
     "@guardian/src-text-area": "^2.8.1",
     "@guardian/src-text-input": "^2.7.1",
     "@guardian/types": "github:guardian/types#semver:1.1.0",
+    "@guardian/renditions": "git+https://github.com/guardian/renditions.git#0.1.0",
```

Comment:
Why `git+` rather than `github:` out of curiosity?

Reply:
Afraid I'm not sure on that. Think this would be a question for @faresite ?